docs/architecture/adr-020-protobuf-transaction-encoding.md

```diff
@@ -73,27 +74,57 @@ and includes all the core field members that are common across all transaction t
 Developers do not have to include `StdTxBase` if they wish, so it is meant to be
 used as an auxiliary type.
 
-### Signing
+### Signing & Transaction Submission
```

Comment:
What is transaction submission? You mean broadcasting?

Reply:
Yes broadcasting

Reply:
I'd remove every mention of submissions in favor of broadcast(ing).